coremltools/converters/mil/mil/passes/test_use_reflection_padding.py

```diff
@@ -207,3 +214,22 @@ def prog(x1):
             inputs,
             expected_output_shapes={block.outputs[0].name: (1, 2, 6, 10), block.outputs[1].name: (1, 2, 6, 1)},
         )
+
+
+    def test_concat_input_only(self):
+        @mb.program(input_specs=[mb.TensorSpec(shape=(1, 2, 6, 8))])
+        def prog(x):
+            x = mb.concat(values=[x, x, x], axis=0)
```

Comment:
I'm wondering if worst case scenario is covered simply testing on axis 0 or if it need to be tested also on other axis

Reply:
The value of `axis` is independent of the bug.